bundles/AdminBundle/public/js/pimcore/settings/user/panel.js

```diff
@@ -55,6 +55,18 @@ pimcore.settings.user.panel = Class.create(pimcore.settings.user.panels.abstract
                 }
             });
 
+            const treeToolbarItems = [
+                "->"
+            ];
+
+            if(pimcore.globalmanager.get('searchImplementationRegistry').hasImplementation()) {
```

Comment:
this is about user search, which has nothing to do with backend search, right? ... so these changes are not necessary and wrong

Reply:
done 42fb37899687c688bbab359391f08fbd3eb23a4e